_log_cache.html.md.erb

```diff
@@ -1 +1 @@
-Log Cache allows you to view logs and metrics over a specified period of time. The Log Cache includes API endpoints and a CLI plugin to query and filter logs and metrics. To download the Log Cache CLI plugin, see [Cloud Foundry Plugins](https://plugins.cloudfoundry.org/). The Log Cache API endpoints are available by default. For more information about using the Log Cache API, see [Log Cache](https://github.com/cloudfoundry/log-cache#restful-api-via-gateway) on GitHub.
\ No newline at end of file
+Log Cache allows you to view logs and metrics over a specified period of time. The Log Cache includes API endpoints and a CLI plugin to query and filter logs and metrics. To download the Log Cache CLI plugin, see [Cloud Foundry Plugins](https://plugins.cloudfoundry.org/). The Log Cache API endpoints are available by default. For more information about using the Log Cache API, see [Log Cache](https://github.com/cloudfoundry/log-cache-release/tree/main/src#apis) on GitHub.
```

Comment:
Maybe instead of "Log Cache allows you to view..." something like  "Log Cache allows the user to view...", "Log Cache allows viewing of..." or "Log Cache enables viewing of..." can be used in order to avoid addressing the users (CF app owners or operators) directly.